docs/2_2_common_mechanisms.adoc

```diff
@@ -817,7 +817,7 @@ _An example is provided in <<example-scheduled-execution>>.]_
 
 ===== Model Partitions and Clocked Variables [[clocked-variable]]
 
-Each Clock latexmath:[\mathbf{k}] induces a discrete system.
+Each Clock latexmath:[\mathbf{k}] represents a discrete system.
```

Comment:
@andreas-junghanns I thought about that sentence again and still don't see how an output clock can represent a model partition. If it can, how can a model partition represented by an output clock be activated? An output clock can only tick and activate a model partition represented by a connected input clock (preferably of another FMU).